lib/rubocop/cop/style/method_call_parentheses.rb

```diff
@@ -45,19 +55,19 @@ def same_name_assignment?(node)
             if asgn_node.or_asgn_type? || asgn_node.and_asgn_type? ||
                asgn_node.op_asgn_type?
               asgn_node, _value = *asgn_node
-              return false if asgn_node.send_type?
+              next if asgn_node.send_type?
             end
 
-            if asgn_node.masgn_type?
-              mlhs_node, _mrhs_node = *asgn_node
-              var_nodes = *mlhs_node
-              vars = var_nodes.map { |n| (_b = *n).first }
+            yield asgn_node
+          end
+        end
 
-              return vars.include?(method_name)
-            end
+        def variable_in_mass_assignment?(variable_name, node)
+          mlhs_node, _mrhs_node = *node
+          var_nodes = *mlhs_node
+          vars = var_nodes.map { |n| (_b = *n).first }
```

Comment:
Thanks for decrypting it. :) I was happy with just extracting the method at this point. Will be changed.